src/tools/rustdoc-themes/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "rustdoc-themes"
 version = "0.1.0"
-edition = "2018"
+edition = "2021"
```

Comment:
I didn't run `-D rust-2021-compatibility` for this tool because I wasn't sure how to build it. It's unclear to me if this tool is even used.

The source code is pretty tiny (only ~70 lines). Is there a way to run `-D rust-2021-compatibility` with it, or is it unnecessary?

Reply:
I think this should be part of bootstrap; you can add #![deny(...)] at the root if you really want but bootstrap seems better since you don't have to do it in each crate. Can you do that as part of this change?

Reply:
I'm working on a patch that will integrate cargo fix --edition into bootstrap, which should make this unnecessary. We don't need to deny the compatibility lint after migration.

Reply:
So do you want me to do something with the `rustdoc-themes` crate to check the migration?